crates/bevy_ui/src/lib.rs

```diff
@@ -119,7 +121,10 @@ impl Plugin for UiPlugin {
             .register_type::<widget::Label>()
             .add_systems(
                 PreUpdate,
-                ui_focus_system.in_set(UiSystem::Focus).after(InputSystem),
+                (
+                    ui_focus_system.in_set(UiSystem::Focus).after(InputSystem),
+                    ui_windows_system,
```

Comment:
```suggestion
                    ui_windows_system.in_set(UiSystem::Window),
```

I'm not sure if there are other missing ordering constraints, but this is definitely implied by the doc strings above.

Reply:
Ah yeah, I wasn't sure whether to add a label to `UiSet` or not. `ui_window_system` runs in `PreUpdate` so there shouldn't be any need for ordering constraints.